### ISSUE TITLE
Some sr-only texts not localized

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14486]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

Many translations are missing in the distribution translation file for Swedish (for example: the entire area with `alias="visuallyHiddenTexts"`). We have been able to fix much of this by adding a translated version of this area to the Config\Lang\sv-SE.user.xml. That way, we have been able to greatly improve the screen reader UX for non-sighted Swedish speaking users. But som important texts in the Umbraco user interface are still presented with their English language defaults. This is a big problem for our users. Especially as they are often pronounced according to the pronounciation rules of the language of the Umbraco site (in our case Swedish). It turns out that these texts have not been localized.

### Specifics

For example, the file /src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree.html has several non-localized instances of the following text attribute:
`text="Open context node for {{tree.name}}"`

And in the file  /src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree-item.html it is obvious that developers are aware that localisation remains to be done:

```
 <!-- TODO: Use localization for this text -->
            <span class="sr-only">Expand child items for {{node.name}}</span>
```

### Steps to reproduce

Log in to a non-English instance of Umbraco. Use a screen reader. Navigate to the content tree. Listen to buttons such as "Open context menu for..." and "Expand child items for...". You will hear the English language defaults, even if you have added localized strings to translation files. 

### Expected result / actual result

As a user with a screen reader, I would very much prefer to hear button functions and other labels in the language of the Umbraco instance, rather than the default English text.

**Comment by plannero** [comment]
Here, by the way, are translations into Swedish that you might want to add to the Swedish language default translation file. 
```
<area alias="visuallyHiddenTexts">
    <key alias="goBack">Backa</key>
    <key alias="activeListLayout">Aktiv layout:</key>
    <key alias="jumpTo">Hoppa till</key>
    <key alias="group">grupp</key>
    <key alias="passed">godkänd</key>
    <key alias="warning">varning</key>
    <key alias="failed">underkänd</key>
    <key alias="suggestion">förslag</key>
    <key alias="checkPassed">Godkänd</key>
    <key alias="checkFailed">Underkänd</key>
    <key alias="openBackofficeSearch">Öppna sökfunktion (backoffice)</key>
    <key alias="openCloseBackofficeHelp">Öppna/stäng hjälpfunktion</key>
    <key alias="openCloseBackofficeProfileOptions">Öppna/stäng personliga inställningar</key>
    <key alias="assignDomainDescription">Redigera språk och värdnamn för %0%</key>
    <key alias="createDescription">Skapa en ny nod under %0%</key>
    <key alias="protectDescription">Ändra behörigheter för %0%</key>
    <key alias="rightsDescription">Redigera behörigheter för %0%</key>
    <key alias="sortDescription">Ändra sortering av %0%</key>
    <key alias="createblueprintDescription">Skapa innehållsmall baserad på %0%</key>
    <key alias="openContextMenu">Öppna kontextmeny för </key>
    <key alias="currentLanguage">Aktuellt språk</key>
    <key alias="switchLanguage">Byt språk till</key>
    <key alias="createNewFolder">Skapa ny mapp</key>
    <key alias="newPartialView">Del av vy</key>
    <key alias="newPartialViewMacro">Del av vy (makro)</key>
    <key alias="newMember">Medlem</key>
    <key alias="newDataType">Datatyp</key>
    <key alias="redirectDashboardSearchLabel">Sök bland omdirigeringar</key>
    <key alias="userGroupSearchLabel">Sök bland användargrupper</key>
    <key alias="userSearchLabel">Sök bland användare</key>
    <key alias="createItem">Skapa post</key>
    <key alias="create">Skapa</key>
    <key alias="edit">Redigera</key>
    <key alias="name">Namn</key>
    <key alias="addNewRow">Lägg till ny rad</key>
    <key alias="tabExpand">Visa fler alternativ</key>
    <key alias="searchOverlayTitle">Sök i Umbraco backoffice</key>
    <key alias="searchOverlayDescription">Sök efter innehåll, media etc i hela Umbraco.</key>
    <key alias="searchInputDescription">När det finns automatförslag, använd pil upp eller ner, eller använd tabbtangenten. Använd enter för att välja.
    </key>
    <key alias="path">Sökväg:</key>
    <key alias="foundIn">Hittad i</key>
    <key alias="hasTranslation">Har översättning</key>
    <key alias="noTranslation">Saknar översättning</key>
    <key alias="dictionaryListCaption">Post i ordlista</key>
    <key alias="contextMenuDescription">Välj ett av alternativen för att redigera noden.</key>
    <key alias="contextDialogDescription">Utför %0% på noden %1%</key>
    <key alias="addImageCaption">Lägg till bildtext</key>
    <key alias="searchContentTree">Sök i innehållsträdet</key>
    <key alias="maxAmount">Maximalt värde</key>
  </area>
```

**Comment by andr317c** [comment]
Hey!
Thanks a lot for posting this issue. This is something we would like help with. I will mark the issue as up for grabs!

The issue is that we are missing localization in src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree-item.html as well as src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree.html. 😄 

We would also be happy if you have the time to create community PR where you add the Swedish translations listed above into the translation file, it's located in src/Umbraco.Core/EmbeddedResources/Lang/sv.xml
